lib/compiler.js

```diff
@@ -68,6 +69,7 @@ module.exports = function compile(options) {
   resolveRelativePaths(bootScripts, appRootDir);
 
   bootDirs.forEach(function(dir) {
+    dir = path.resolve(dir);
```

Comment:
This looks like an unrelated change to me, why is it needed by mixins? Could you please revert?